performance.rst

```diff
@@ -211,21 +211,24 @@ deployment process too):
 
 .. _profiling-applications:
 
-Profiling Applications
-----------------------
+Profiling Symfony Applications
+------------------------------
+
+Profiling with Blackfire
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 `Blackfire`_ is the best tool to profile and optimize performance of Symfony
 applications during development, test and production. It's a commercial service,
 but provides free features that you can use to find bottlenecks in your projects.
 
+Profilwing with Symfony Stopwatch
```

Comment:
Typo -w :)

Reply:
Also what about adding a ref to the profiler/wdtoolbar on dev env? I think it can make a bridge with adding stopwatch are the profiler leverage the stopwatch